bin/ruby-interview

```diff
@@ -1,4 +1,6 @@
 #!/usr/bin/env ruby
 # frozen_string_literal: true
 
-puts 'Good luck!'
\ No newline at end of file
+require_relative '../lib/data_processor'
+
+DataProcessor.new(*ARGV).process_data
```

Comment:
What can a DataProcessor do other that _processing data_? 😄

Reply:
What is the reason of choosing to create objects of a class instead of just provide a simple function that takes the CLI arguments and returns the desired result? What is the responsability of this class?

Reply:
This class has too many responsibilities, I will split it